<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectWidgets</class>
 <widget class="QDockWidget" name="InspectWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inspect Widgets</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="inspectButton">
      <property name="toolTip">
       <string>Inspect widgets (can be cancelled by pressing shift)</string>
      </property>
      <property name="text">
       <string>Inspect</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="identifierLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="styleSheetText">
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pathLabel">
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
